homeassistant/components/sensor/__init__.py

```diff
@@ -401,7 +401,10 @@ def state(self) -> Any:
                     )
 
                 if value.tzinfo != timezone.utc:
-                    value = value.astimezone(timezone.utc)
+                    try:
+                        value = value.astimezone(timezone.utc)
+                    except OverflowError:
+                        pass
```

Comment:
Why should we just pass here? Won't that mean that we return a value with incorrect timezone?